src/xdg.rs

```diff
@@ -57,7 +57,9 @@ impl DesktopEntries {
         for dir_entry in dirs.iter().flat_map(|d| fs::read_dir(d.join("applications")).ok()) {
             for desktop_file in dir_entry
                 .filter_map(|entry| entry.ok())
-                .filter(|entry| entry.file_type().map_or(false, |ft| ft.is_file()))
+                .filter(|entry| {
+                    entry.file_type().map_or(false, |ft| ft.is_file() || ft.is_symlink())
```

Comment:
Is this different than `!ft.is_directory()`?

Reply:
Yes, will exclude sockets and such.